dev/com.ibm.ws.channelfw/bnd.bnd

```diff
@@ -28,15 +28,18 @@ IBM-Default-Config: OSGI-INF/wlp/defaultInstances.xml
 Export-Package: com.ibm.websphere.channelfw*;provide:=true, \
 	com.ibm.wsspi.channelfw*;provide:=true, \
 	com.ibm.wsspi.connmgmt;provide:=true, \
-	com.ibm.wsspi.bytebuffer;provide:=true, \
 	com.ibm.wsspi.tcpchannel;provide:=true, \
-    com.ibm.wsspi.udpchannel;provide:=true, \
-    com.ibm.websphere.endpoint;provide:=true, \
-	com.ibm.io.async
+	com.ibm.wsspi.udpchannel;provide:=true, \
+	com.ibm.websphere.endpoint;provide:=true, \
+	com.ibm.io.async, \
+	com.ibm.wsspi.bytebuffer
```

Comment:
the copyright year should be updated for this file as well

Reply:
I have updated the copyright. 
Rather oddly com.ibm.ws.wsbytebuffer is the name of the bundle and the project, here I assume we are both referring to the com.ibm.wsspi.bytebuffer java package. On the topic of whether/how that should appear on the Export-Package, I thought the desired outcome was that the external contract of the channelfw bundle should be unchanged so that it will now import the package from the newly factored out com.ibm.ws.wsbytebuffer bundle and then re-export it to its own existing clients who should not need to know about the refactoring? (Having said that I have had to the com.ibm.ws.wsbytebuffer project name to some of the -buildpaths/-testpaths in bnd files - so I will investigate if that desired outcome is not possible)

Reply:
Discussed, and we believe this is OK.